sample_apps_reinvent2021/go/query-operations.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/timestreamquery"
+	"go_sample/utils"
+
+	"flag"
+	"fmt"
+	"os"
+)
+
+func main() {
+	// setup the query client
+	sess, err := session.NewSession(&aws.Config{Region: aws.String("us-east-1")})
```

Comment:
What about other regions?

Reply:
so currently the samples seems to provide specific region input for ScheduledQuery samples only and the older samples are copied verbatim.    I agree we should make this change - let me take this up.